Mathlib/Algebra/Module/Presentation/Basic.lean

```diff
@@ -162,6 +162,10 @@ lemma fromQuotient_mk (x : relations.G →₀ A) :
 lemma fromQuotient_comp_toQuotient :
     solution.fromQuotient.comp relations.toQuotient = solution.π := rfl
 
+@[simp]
+lemma fromQuotient_toQuotient (x : relations.G →₀ A) :
+    solution.fromQuotient (relations.toQuotient x) = solution.π x := rfl
```

Comment:
If `relations.toQuotient` is the right map instead, we probably could/should drop `fromQuotient_mk`?

Reply:
Yes, I have removed it in https://github.com/leanprover-community/mathlib4/pull/18327/commits/5c18c5e8d91324fc946e5bc0cb45b14253c438bf I had to make `Quotient` a def rather than an abbrev.